.github/workflows/conventional-commits.yml

```diff
@@ -12,4 +12,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: chanzuckerberg/github-actions/.github/actions/conventional-commits@v1.3.2
+      - uses: chanzuckerberg/github-actions/.github/actions/conventional-commits@main
```

Comment:
Since #61 was closed in favor of this PR, I am pulling this change in that was there.